lib/libutee/include/tee_api_defines.h

```diff
@@ -64,6 +64,8 @@
 #define TEE_LOGIN_APPLICATION_USER      0x00000005
 #define TEE_LOGIN_APPLICATION_GROUP     0x00000006
 #define TEE_LOGIN_TRUSTED_APP           0xF0000000
+/* Private login method for REE kernel clients */
+#define TEE_LOGIN_REE_KERNEL		0x80000000
```

Comment:
Shouldn't this be in `<tee_api_defines_extensions.h>` instead?

Reply:
Okay. Should we also move `TEE_LOGIN_TRUSTED_APP` to `<tee_api_defines_extensions.h>`? I guess it is an extension too.

Reply:
No, it isn't. `TEE_LOGIN_TRUSTED_APP` is in table 4-2 of the GP spec.

Reply:
Ah, I see. So its in GP internal API spec. I was looking at GP client API spec.